ed/idlpatches/webcodecs.idl.patch

```diff
@@ -0,0 +1,104 @@
+From 7bb8d539a667ac3145e863c856944e3e862d5b6a Mon Sep 17 00:00:00 2001
+From: Francois Daoust <fd@tidoust.net>
+Date: Mon, 14 Jun 2021 19:01:19 +0200
+Subject: [PATCH] Revert IDL while dictionary as attribute issue gets fixed
+
+https://github.com/w3c/webcodecs/issues/278
+---
+ ed/idl/webcodecs.idl | 47 +++++++++++++++++++-------------------------
+ 1 file changed, 20 insertions(+), 27 deletions(-)
+
+diff --git a/ed/idl/webcodecs.idl b/ed/idl/webcodecs.idl
+index dbb3cc2de..16d6bd5ed 100644
+--- a/ed/idl/webcodecs.idl
++++ b/ed/idl/webcodecs.idl
+@@ -161,7 +161,6 @@ dictionary VideoEncoderConfig {
+   [EnforceRange] unsigned long displayHeight;
+   HardwareAcceleration hardwareAcceleration = "allow";
+   DOMString scalabilityMode;
+-  BitrateMode bitrateMode = "variable";
```

Comment:
should we only revert the bits that are problematic? do we have a policy or precedents?

Reply:
I think I confused the previous commit for webcodecs.idl in webref and the previous commit for the source spec. There have been more than one commit that changes the IDL in the WebCodecs spec between two updates in webref in practice.

Anyway, I rolled back the rollback. The attribute is now preserved.